pages/collections/_type/_.vue

```diff
@@ -1,67 +1,80 @@
 <template>
   <b-container
     data-qa="entity page"
+    fluid
+    class="entity-page"
   >
-    <b-row class="flex-md-row pt-3">
+    <b-row class="flex-md-row pt-5 bg-white mb-4">
       <b-col
         cols="12"
-        md="9"
       >
-        <EntityDetails
-          :attribution="attribution"
-          :depiction="depiction"
-          :description="description"
-          :is-editorial-description="hasEditorialDescription"
-          :title="title"
-          :depiction-link-title="$t('goToRecord')"
-        />
+        <b-container>
+          <EntityDetails
+            :description="description"
+            :is-editorial-description="hasEditorialDescription"
+            :title="title"
+          />
+          <client-only>
+            <h2
+              v-if="relatedEntities || relatedCollectionCards"
+              class="related-heading text-uppercase mb-2"
+            >
+              {{ $t('relatedCollections') }}
+            </h2>
+            <section
+              v-if="relatedEntities"
+              class="mb-2"
+            >
+              <RelatedChip
+                v-for="relatedEntity in relatedEntities"
+                :id="relatedEntity.id"
+                :key="relatedEntity.id"
+                :link-gen="suggestionLinkGen"
+                :title="relatedEntity.prefLabel[$i18n.locale]"
+                :img="`${relatedEntity.isShownBy.thumbnail}&size=w200`"
+                data-qa="related entities"
+              />
+            </section>
+            <section
+              v-else-if="relatedCollectionCards"
+              class="mb-2"
+            >
+              <RelatedChip
```

Comment:
These links do not work. See for example the curated card links at http://localhost:3000/en/collections/topic/55-fashion

Reply:
for some reason on related collection cards the identifier wasn't being sent to linkGen function, I have refactored the linkGen to take in the entire card/item object as an argument and then access the id/identifier from it, links are now working on both related collection cards and related entities.